site/_docs/reference.md

```diff
@@ -2615,6 +2619,7 @@ semantics.
 | p q | DATEADD(timeUnit, integer, datetime)         | Equivalent to `TIMESTAMPADD(timeUnit, integer, datetime)`
 | p q | DATEDIFF(timeUnit, datetime, datetime2)      | Equivalent to `TIMESTAMPDIFF(timeUnit, datetime, datetime2)`
 | q | DATEPART(timeUnit, datetime)                   | Equivalent to `EXTRACT(timeUnit FROM  datetime)`
+| b | DATE_SUB(date, interval int64 date_part)       | Subtracts int64_expression units of date_part from the date, independent of any time zone.
```

Comment:
not `int64_expression units of date_part`; it's just an interval expression

remove '.' from end of line

DATE_SUB, TIME_SUB, TIMESTAMP_SUB should have very similar descriptions to TIMESTAMP_ADD. I dropped the ball when reviewing TIMESTAMP_ADD. It should have been
```
TIMESTAMP_ADD(timestamp, interval) | Returns the TIMESTAMP value that occurs *interval* after *timestamp*
```

`TIMESTAMP_SUB` should be "the TIMESTAMP value that occurs *interval* before *timestamp*", and similarly DATE_SUB, TIME_SUB